h2o-bindings/bin/custom/R/gen_stackedensemble.py

```diff
@@ -23,13 +23,12 @@ def update_param(name, param):
     validate_params="""
 # Get the base models from model IDs (if any) that will be used for constructing model summary
 if(!is.list(base_models) && is.vector(x)) {
-   base_models <- as.list(base_models)
+  if (inherits(base_models, "H2OGrid")) {
+    base_models <- list(base_models)
+  } else {
+    base_models <- as.list(base_models)
+  }
```

Comment:
can be written as a single expression, a bit like in Python: ensures that there'll never be any side-effect sneaking into this logic
```
base_models <- if (inherits(base_models, "H2OGrid")) list(base_models) else as.list(base_models)
```

Reply:
That's a good point. I haven't thought of that :)